starter/script.js

```diff
@@ -1 +1,126 @@
-// put code here!
+confirm("Are you over 18? Clicking OK proves this beyond a doubt. It's science.");
+
+alert("It is a steamy, wind-swept night and you are alone in your grammaw's house. It is boring.");
+
+alert("The internet is out and all she has for entertainment are these crusty cheese-ball romance novels. You have a choice: Go in the living room and help her file her bunions or pretend to read one of these things and let her think you're really into it.");
+
+//==========Night At Grandmaws Function===+ BUNIONS ENDING 1==
+nightAtGrandmas();
+
+function nightAtGrandmas(){
+var BunionsOrRead = prompt("Now, what will it be sweet, honey-child? Ol' Mammaw's raspy bunions or a skanky bodice ripper? Peck in 'B' for 'Bunions' or 'R' for 'Read the damn book'.").toUpperCase();
+      if(BunionsOrRead === "B"){
+        alert("You chose your fate, bro. Damn. Go get the cheese-grater and the lotion! THE END FOR YOU!");
+      }
+      else if (BunionsOrRead === "R"){
+        book();
+  }
+}
```

Comment:
What if the user does not submit "B" or "R"? What kind of code could you put in an `else` statement that would re-prompt the user?